src/tests/ftest/util/dmg_utils.py

```diff
@@ -85,6 +85,13 @@ class DmgCommand(DmgCommandBase):
             r"Targets:\[[0-9 ]+\]\s+Rank:\d+\s+State:(\w+))",
     }
 
+    def _get_json_result(self, sub_command_list=None, **kwargs):
```

Comment:
Perhaps we can use a single _get_result() function that checks if the json output flag was set or not. This if statement was removed from down below and we want to be able to use this test utility to output in json or plain text. The other modules in this file will be updated to make use of this function also -- likely in DAOS-4908, which I have put on hold until this ticket lands. @mjmac  @phender

Reply:
My vote would be for keeping the `_get_json_result()` - with the addition of returning `json.loads(self.result.stdout)` - and eventually using it to replace the `_get_result()` method calls in all the dmg command helper methods.  By not overriding the `_get_result()` method we leave the option open to still get non-json output if we ever need it.

Reply:
Yep. Forgot to make this change when I pushed the latest update. I'll push it after dealing with any other review feedback.